docs/add-a-system-config.rst

```diff
@@ -334,9 +334,11 @@ available on your system.
 
 .. note::
 
-    For ``mpi``, you need to define ``"mpi": {"buildable": False},`` as a virtual
-    package, and then define your MPI package as we have for the ``openmpi`` package.
-    This is to ensure Spack uses our MPI, and does not try to build another MPI package.
+    For packages that declare virtual dependencies, e.g. ``depends_on("mpi")``, you need
+    to define ``"mpi": {"buildable": False},`` as a virtual package, and then define
+    your MPI package as we have for the ``openmpi`` package. This is to ensure Spack
+    uses our MPI, and does not try to build another MPI package. See a similar example
+    for ``blas``, ``lapack``, and their provider ``atlas``.
```

Comment:
@scheibelp  Is this statement true or would you word this differently. I updated it because I added blas and lapack virtuals as well.